distribution/src/bin/elasticsearch.bat

```diff
@@ -5,6 +5,7 @@ setlocal enableextensions
 
 SET params='%*'
 SET checkpassword=Y
+SET enrolltocluster=N
```

Comment:
We don't appear to be setting or using this.

Reply:
:doh